test/pageobjects/CrossDeviceMobileNotificationSent.js

```diff
@@ -50,6 +51,10 @@ class CrossDeviceMobileNotificationSent extends BasePage {
   async clickResendLink() {
     this.resendLink.click()
   }
+
+  async waitForYourMobilePhoneIconToBeLocated() {
+    this.driver.wait(until.elementLocated(By.css('.onfido-sdk-ui-Theme-icon')))
```

Comment:
shouldn't this be using `yourMobilePhoneIcon` ? like this is redundant

Reply:
In the beginning, I tried that 

`this.driver.wait(until.elementLocated(By.css(this.yourMobilePhoneIcon)))`

but Selenium expects String as a value and throws:

`InvalidArgumentError: invalid argument: 'value' must be a string` hence I use the selector directly.

Reply:
but it's not really a string, because you are passing the output of `By.css(string)`, isn't it?

Reply:
I checked and the difference is that $ actually returns `driver.findElement(By.css())` instead of just `By.css(selector)`. if you are going to share the selectors I guess you can have it as a property called something like `[elementName]Selector`?

Reply:
e.g.
`get yourMobilePhoneIconSelector() { return By.css('.onfido-sdk-ui-Theme-icon')}`
`get yourMobilePhoneIcon() { return this.$(this.yourMobilePhoneIconSelector)}`

Reply:
Changed in 3f9aac2